mica-webapp/src/main/java/org/obiba/mica/web/controller/CartController.java

```diff
@@ -33,4 +33,27 @@ public ModelAndView get() {
     }
   }
 
+  @GetMapping("/lists")
```

Comment:
There is no template for this one. And is it necessary to have a list of lists? Isn't it enough to have them in the sidebar (with the count of documents if you wish)?